lutris/util/wine/proton.py

```diff
@@ -52,35 +52,12 @@ def get_umu_path() -> str:
     The path that this returns will be considered an Umu path by is_umu_path().
 
     If this script can't be found this will raise MissingExecutableError."""
-
-    # We can access Umu via it's main .py file, or via the public name which is just
-    # symbolic link to the same file. We'll check both for compatibility, just in case
-    # Umu gets translated into Perl or something.
-    entry_points = ["umu_run.py", "umu-run"]
-
-    custom_path = settings.read_setting("umu_path")
-    if custom_path:
-        for entry_point in entry_points:
-            entry_path = os.path.join(custom_path, entry_point)
-            if system.path_exists(entry_path):
-                return entry_path
-
-    # We only use 'umu-run' when searching the path since that's the command
-    # line entry point.
     if system.can_find_executable("umu-run"):
         return system.find_required_executable("umu-run")
-    path_candidates = (
-        "/app/share",  # prioritize flatpak due to non-rolling release distros
-        "/usr/local/share",
-        "/usr/share",
-        "/opt",
-        settings.RUNTIME_DIR,
-    )
-    for path_candidate in path_candidates:
-        for entry_point in entry_points:
-            entry_path = os.path.join(path_candidate, "umu", entry_point)
-            if system.path_exists(entry_path):
-                return entry_path
+
+    if os.path.exists(entry_point := f"{settings.RUNTIME_DIR}/umu/umu-run"):
```

Comment:
The walrus operator (:=) was introduced in Python 3.8, and we support by back to 3.7. You can't use this feature. Sorry.

Also, I think system.path_exists() should be used here since it supports '~'.

Also also, os.path.join() should probably be preferred over using an f-string. Though maybe that just my Windows experience talking, where paths are extra crazy.

Reply:
I'll revert the syntax as I wasn't aware and will make the changes. Also, if that's the case, is there any specific reason why the Python version defined in the project's pyproject.toml was not downgraded to reflect that?

Reply:
Hmm, I see what you mean. This is surprise to me, and it arrived in a seemingly unrelated PR #5237. I'll ping @strycore about this on Discord. Maybe we're okay with the walrus now and I'm the one whose out of date here.

Reply:
OK, he's told me that we're go for 3.8, so the walrus is our friend and you don't need to change this.